Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -23,52 +23,64 @@ private typealias CategoryNumber = Int
 private typealias CategorySpec = (section: SectionNumber?, rows: Int, offset: Int)
 
 private struct HistoryPanelUX {
-    static let WelcomeScreenPadding: CGFloat = 15
-    static let WelcomeScreenItemTextColor = UIColor.grayColor()
-    static let WelcomeScreenItemWidth = 170
+    private static let WelcomeScreenItemTextColor = UIColor.grayColor()
+    private static let WelcomeScreenItemWidth = 170
+    private static let SyncedTabsCellChevronInset: CGFloat = 20
+    private static let SyncedTabsCellChevronSize: CGFloat = 20
+    private static let SyncedTabsCellImageSize: CGFloat = 32
+    private static let SyncedTabsCellHeight: CGFloat = 60
+    private static let SyncedTabsCellChevronLineWidth: CGFloat = 4.0
+    private static let SyncedTabsCellTextColor = UIColor(red: 92/255, green: 92/255, blue: 92/255, alpha: 1.0)
 }
 
-class HistoryPanel: SiteTableViewController, HomePanel {
+class HistoryPanel: UIViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
-    private lazy var emptyStateOverlayView: UIView = self.createEmptyStateOverview()
-
-    private let QueryLimit = 100
-    private let NumSections = 4
-    private let Today = getDate(dayOffset: 0)
-    private let Yesterday = getDate(dayOffset: -1)
-    private let ThisWeek = getDate(dayOffset: -7)
-
-    // Category number (index) -> (UI section, row count, cursor offset).
-    private var categories: [CategorySpec] = [CategorySpec]()
-
-    // Reverse lookup from UI section to data category.
-    private var sectionLookup = [SectionNumber: CategoryNumber]()
-
-    var refreshControl: UIRefreshControl?
+    var profile: Profile!
+    private lazy var tableViewController: HistoryPanelViewController = {
+        return HistoryPanelViewController()
+    }()
+    private lazy var syncedTabsHeaderCell: SyncedTabsTableViewCell = {
+        return SyncedTabsTableViewCell()
+    }()
 
     init() {
         super.init(nibName: nil, bundle: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(HistoryPanel.notificationReceived(_:)), name: NotificationFirefoxAccountChanged, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(HistoryPanel.notificationReceived(_:)), name: NotificationPrivateDataClearedHistory, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(HistoryPanel.notificationReceived(_:)), name: NotificationDynamicFontChanged, object: nil)
+
+        self.tableViewController.homePanelDelegate = homePanelDelegate
     }
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.tableView.accessibilityIdentifier = "History List"
-    }
 
-    override func viewWillAppear(animated: Bool) {
-        super.viewWillAppear(animated)
+        self.tableViewController.tableView.accessibilityIdentifier = "History List"
```

Comment:
Nit: `HistoryPanelViewController` should be responsible for setting it's table's accessibility identifier instead of the parent VC.